packages/synapse-react-client/src/components/DirectDownload/DirectDownload.tsx

```diff
@@ -36,7 +36,10 @@ type DirectDownloadIconProps = {
   iconSvgPropOverrides?: Partial<IconSvgProps>
 }
 
-function DirectDownloadIcon(props: DirectDownloadIconProps) {
+const DirectDownloadIcon = React.forwardRef<
```

Comment:
Needed to add `forwardRef` to properly wrap this component in a `Tooltip` without a separate wrapper (see below)